seda.muradyan/repo/Lesson_3/exercise_09/exercise_09.py

```diff
@@ -0,0 +1,10 @@
+# Home work lesson 3/9
+
+price = float(input ("Enter price of the meal: "))
+tipPracent = float(input ("Enter tip percent: "))
```

Comment:
CSW: Variable names in python should be lower case and separated with an underscore.